src/Sylius/Bundle/AdminBundle/Controller/DashboardController.php

```diff
@@ -33,6 +33,9 @@ public function __construct(
         private ?SalesDataProviderInterface $salesDataProvider = null,
         private ?StatisticsDataProviderInterface $statisticsDataProvider = null
```

Comment:
Probably we should place both services into one argument and provide proper null handling